### DISCUSSION TITLE
Error Not Finding the Json file after successfully adding adding feed to library

### DISCUSSION THREAD

**Discussion by HSBNDev** [rrirower/Channel-Feed-Manager#123] [Q&A]
I downloaded the latest release of the feed manager 1.0.0.9. I downloaded my json feed into a .json file and added it to my library using the "Add to your feed library" button. Which showed a success message "Successfully merged 1 feed file(s) into your channel library" and it appears in the list box on the left side of the buttons. When i double click the feed, it then shows an error message that "The specified file "C:\Users\<My Username>\Onedrive\Documents\Channel Feed Manager\Channels\Channel feeds\<File Name>.json was not found" followed by a message "Before you go.... Your feed file has been modified. Do you want to save the changes before exiting?"

I have used the uninstaller to uninstall the package and deleted the ..\Channel Feed Manager\Channels\Channel feeds directory, and then reinstalling the application and i keep getting the same error. 

I had first run the application a few days ago that successfully read my feed and provided error messages in the log stating that the "rating" field was missing from my feed. After adding the rating field to my feed it is now populating the error with the new feed and wont load it any longer.

**Comment by rrirower** [comment]
Can you provide either a URL to the feed file or, the actual feed file for testing?

**Reply by HSBNDev** [reply]
Here is the feed file

[roku5.json](https://github.com/rrirower/Channel-Feed-Manager/files/13757085/roku5.json)

**Comment by rrirower** [comment]
Resolved in 1.0.1.0
